frontend/public/components/monitoring/silence-form.tsx

```diff
@@ -336,7 +354,15 @@ const SilenceForm_: React.FC<SilenceFormProps> = ({ defaults, Info, title }) =>
                         onChange={(e) => setMatcherField(i, 'isEqual', !e.currentTarget.checked)}
                         type="checkbox"
                       />
-                      &nbsp; {t('public~Negative matcher')}
+                      &nbsp;{' '}
+                      <Popover
```

Comment:
Does this break clicking on the label to toggle the checkbox? If so, it might be an accessibility issue.